packages/client/manifest.json

```diff
@@ -5186,5 +5186,88 @@
         "suffix": "repeater"
       }
     ]
+  },
+  "rowexplorer": {
+    "block": true,
+    "name": "Row explorer block",
+    "icon": "PersonalizationField",
+    "size": {
+      "width": 100,
+      "height": 100
+    },
+    "settings": [
+      {
+        "type": "dataSource",
```

Comment:
We should update this to be of type "table" (and name it "Table"), since this will only work with tables.